tests/phpunit/CRM/Member/Form/MembershipTest.php

```diff
@@ -244,9 +241,9 @@ public function testFormRuleRollingLifetimeEnd() {
     $unixNow = time();
     $unixYearFromNow = $unixNow + (365 * 24 * 60 * 60);
     $params = array(
-      'join_date' => date('m/d/Y', $unixNow),
-      'start_date' => date('m/d/Y', $unixNow),
-      'end_date' => date('m/d/Y',
+      'join_date' =>  date('Y-m-d'),
```

Comment:
@eileenmcnaughton why the extra space here?

Reply:
mostly to upset jenkins...